tests/src/PublicClass.php

```diff
@@ -0,0 +1,11 @@
+<?php
+declare(strict_types = 1);
+
+namespace PrivateModule;
+
+use Attributes\AttributeClass;
+
+#[AttributeClass]
+interface PublicClass
```

Comment:
Can you make this one a class so that both interfaces and classes are tested?

Reply:
Done